README.md

```diff
@@ -27,11 +27,18 @@ west zephyr-export
 cd aos_core_zephyr
 
 west build -b ${BOARD} -p auto
+west build -t fix_dtb
```

Comment:
It looks like some kind of WA, doesn't it? Please add explanation what this target does and when we need to use it.